xc/device.ml

```diff
@@ -1746,7 +1746,8 @@ let start_vgpu ~xs task ?restore_fd domid vgpus vcpus =
 					[uuid, fd]
 			in
 			let args = vgpu_args_of_nvidia domid vcpus vgpu fds in
-			let ready_path = Printf.sprintf "/local/domain/%d/vgpu-pid" domid in
+			let ready_path = Printf.sprintf "/local/domain/%d/vgpu/state" domid in
+			let ready_val = if restore_fd = None then "running" else "restoring" in
```

Comment:
What is going to read this path? Can we define the path and the valid values in a single place, shared by this code and the reader?